src/store/store.ts

```diff
@@ -19,12 +19,42 @@ import {
   UpdateStatus,
 } from "common/models";
 
-const InitialStatuses = {
-  New: { id: 0, name: "new", isDefault: true, isDeleted: false },
-  Commited: { id: 1, name: "commited", isDefault: false, isDeleted: false },
-  InProgress: { id: 2, name: "in progress", isDefault: false, isDeleted: false },
-  Done: { id: 3, name: "done", isDefault: false, isDeleted: false },
-  Fixed: { id: 4, name: "fixed", isDefault: false, isDeleted: false },
+
+export const deleteStatus = (state: RootState, action: DeleteStatus) => {
+  const statusId = action.payload;
+  const statusToDeleteIndex = state.statuses.findIndex(
+    (status) => status.id === statusId
+  );
+
+  if (statusToDeleteIndex === -1) {
+    throw new Error('Status does not exist');
```

Comment:
It is useful to provide what status we are trying to delete in the error message
**Status with id 42 does not exist**

Reply:
done